core/bubble.js

```diff
@@ -144,6 +145,17 @@ Blockly.Bubble.unbindDragEvents_ = function() {
   }
 };
 
+/*
+ * Handle a mouse-up event while dragging a bubble's border or resize handle.
+ * @param {!Event} e Mouse up event.
+ * @private
+ */
+Blockly.Bubble.bubbleMouseUp_ = function(/*e*/) {
```

Comment:
Hm.  It's going to be called with the e, even if we just drop it on the floor.  Getting rid of it makes it harder to know that it's possible to extend this to use the event, but uncommenting it makes eslint sad.  I think eslint is okay with the current setup.

Alternately, could I delete the @param and leave a note that an event e may be passed, without formalizing it with the @param tag?